libs/platform/auth/src/bapi/feature.ts

```diff
@@ -34,7 +34,7 @@ export class BapiAuthFeature extends OauthFeature implements AppFeature {
       providers: [
         {
           id: 'spryker',
-          clientId: 'frontend',
```

Comment:
Should we set `frontend` as a default one (instead of an empty string) to make tests work?